tests/c/setlongjmp.c

```diff
@@ -0,0 +1,48 @@
+// Run-time:
+//   env-var: YKD_PRINT_IR=jit-pre-opt
+//   env-var: YKD_SERIALISE_COMPILATION=1
+//   env-var: YKD_PRINT_JITSTATE=1
+//   stderr:
+//     ...
+//     jit-state: stop-tracing
+//     jit-state: invalid trace
```

Comment:
I think all the jit states should be hyphenated for consistency, but also see comment below about the reason for aborting.

Reply:
[44e67a7](https://github.com/ykjit/yk/pull/609/commits/44e67a77a582bd8919a16ad752e73f2451d81c07)

Reply:
[875f484](https://github.com/ykjit/yk/pull/609/commits/875f484321f343f346c606a8fa577838b041f81d)